crates/mqtt-format/src/v5/packets/suback.rs

```diff
@@ -32,6 +32,13 @@ crate::v5::reason_code::make_combined_reason_code! {
     }
 }
 
+impl SubackReasonCode {
+    /// Return whether reason code is granted
+    pub fn is_granted(&self) -> bool {
+        *self == Self::GrantedQoS0 || *self == Self::GrantedQoS1 || *self == Self::GrantedQoS2
+    }
+}
```

Comment:
Could you make it return an Option<QoS>? This way users don't lose out on this IMO important info.

One can still choose to quickly check if something got granted with `granted().is_some()` for example

Reply:
Oh, that's a nice idea as well! Will do.